package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "prepare": "husky install",
     "compile-assets": "grunt",
-    "watch-node": "ts-node-dev -r dotenv/config --watch .env --respawn server.ts",
+    "watch-node": "nodemon -r dotenv/config ./server.ts",
```

Comment:
Since switching to `ts-node-dev` I've noticed a few issues with it failing to restart and occasionally throwing type errors that won’t resolve. When I added `ts-node-dev` I was a bit worried about stability since I’ve read reports of other people having occasional problems with it, but since it was much faster at server restarts I thought it was worth giving a go.

I have tried to debug the issues I’ve been having with `ts-node-dev`, but haven’t had much success so for stability I think it's worth switching back to `nodemon`. However, since `nodemon` is now executing `server.ts` directly with `ts-node` instead of first compiling to .js startup times should be about equal to `ts-node-dev`, but maybe 2-3 seconds slower on reloads.

.njk changes should still load immediately, and the asset build is unchanged. I've also made a small change to watch for changes in the "integration_tests" dir. This will allow `nodemon` to reload when using `start-feature:dev` and updating .json fixtures for integration tests.